dll/win32/shell32/dialogs/drvdefext.cpp

```diff
@@ -238,6 +238,7 @@ CDrvDefExt::PaintStaticControls(HWND hwndDlg, LPDRAWITEMSTRUCT pDrawItem)
 
             MoveToEx(pDrawItem->hDC, x, y, NULL);
             LineTo(pDrawItem->hDC, x, y + 10);
+            SetPixel(pDrawItem->hDC, x, y + 10, RGB(0, 0, 0));
```

Comment:
I'm sorry but this is still kind of a hack. Looking at your With_SetPixel image (zoomed in for more clarity):

![image](https://user-images.githubusercontent.com/1969829/232151264-3df9cda9-bcde-46b3-a0ae-4a2821751497.png)

Also, how is the top border of the pie chart being able to be drawn with the correct black color? I don't see where that black color gets selected in the DC.

Reply:
The brush is selected on line 244: `SelectObject(pDrawItem->hDC, hOldPen);` where it's restoring the default DC pen backed up earlier on line 221 `HPEN hOldPen = (HPEN)SelectObject(pDrawItem->hDC, hDarkMagPen);`.  According to [Pie](https://learn.microsoft.com/en-us/windows/win32/api/wingdi/nf-wingdi-pie) on MSDN "The pie is outlined by using the current pen and filled by using the current brush."

Reply:
What if the default DC pen is not black?

Reply:
That was how it was originally in the code before my changes, should I change it to explicitly set the pen?

Reply:
Changed it to explicitly set the pen instead of relying on whatever the default color may be.

Reply:
I am still unsure why you need a `SetPixel` directly after drawing a line, that seems like a hack to me.

Reply:
The Line draws the ~10 pixel pie chart height, in magenta.

Reply:
But setpixel is not drawing a line, SetPixel is just updating one pixel.

Reply:
![image](https://user-images.githubusercontent.com/1969829/235143163-22471b66-33cc-4f74-b804-ed0068f160d4.png)